pandas/core/arrays/numpy_.py

```diff
@@ -376,7 +376,8 @@ def _cmp_method(self, other, op):
             other = other._ndarray
 
         pd_op = ops.get_array_op(op)
-        result = pd_op(self._ndarray, other)
+        with np.errstate(all="ignore"):
+            result = pd_op(self._ndarray, other)
```

Comment:
BTW, we probably don't need to add this one here, since when going through a DataFrame/Series op, this will also already be catched on that level (and be don't consistently catch it in other EA ops). But this keeps current behaviour (and this not used in practice anyway)